app/models/application_form.rb

```diff
@@ -436,6 +443,14 @@ def self.with_unsafe_application_choice_touches
     return_value
   end
 
+  def show_new_reference_flow?
```

Comment:
Nitpick: models really don't know anything about showing or hiding. Not a blocker.